python/parser.js

```diff
@@ -11,8 +11,10 @@
     }
     let element = ''
     let options = ''
-    if (cs.frames === undefined && cs.region === undefined) element = '.window()'
-    else {
+    element = '.window()'
+    if (cs.scrollRootElement) element += `.scroll_root_element(${printSelector(cs.scrollRootElement)})`
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
    if (cs.scrollRootElement) {element += `.scroll_root_element(${printSelector(cs.scrollRootElement)})`}
```